api/BudgetState.json

```diff
@@ -29,7 +29,7 @@
             "version_added": null
           },
```

Comment:
This is a an API from an obsolete draft spec which probably should not have been implemented out of a flag, do we have this in our browser? 

Spec: https://wicg.github.io/budget-api/#budget-state-interface
MDN: https://developer.mozilla.org/en-US/docs/Web/API/BudgetState

Reply:
I have confirmed we do not support this
![image](https://user-images.githubusercontent.com/4225330/67208926-55571280-f40e-11e9-8d5a-25191fe4bb02.png)